### PR TITLE
[ZH] translation of 06-2 and 06.md

### PR DESCRIPTION
**Pull request by JonathanSum**
I have translated the top 50% of the RNN(06-02) in Chinese. 

I passed the course on deep learning.ai and I also wrote a few notebooks to help students in the coursera Tensorflow time series seq2seq notebook.

### CODE REVIEW

**Comment by Atcold** [comment]
What is this stuff???

Screenshot 2020-04-16 at 15 27 19

**Comment by JonathanSum** [comment]
Sorry, I uploaded my files to the wrong location. It was the text file for training a GPT2 for different projects.

**Comment by JonathanSum** [comment]
The [ZH] translation of 06-2(100%) will be uploaded in 1 to 2 hours.

**Comment by JonathanSum** [comment]
@Atcold , I saw this on the English version:

In attention, to produce the current word at each time step...

Shouldn't the 'a' in attention to be capital?

**Comment by JonathanSum** [comment]
Just a comment, Maybe in the future I will still edit my translation in free time and hope someone improves the translation if it is needed. In addition, I have also added the picture for week 6-02 in Chinese because I hope the Chinese version is for people who can barely read English or do not know English at all.

**Comment by JonathanSum** [comment]
Just a little comment. Once again, thank you for sharing the course material, including video, homework, exam, and more. Thank You.

**Comment by Atcold** [comment]
Why did you upload all these images?
We're going to use the same for all the translations.

And you're welcome.
We have a group of people looking after the grammar for the English part :)

**Comment by JonathanSum** [comment]
I was thinking that I should translate it as much as possible for non-English speakers.
It is not everyone in the world who speaks English or smart enough to quickly understand English parts. When they see English, it is just like forcing their mind to stop or wasting their time to search on the dictionary to translate it by themselves, especially for people who are not very smart. In addition, this course is for everyone, and that is one of the reasons why we translate it.

Thus, I translated those images too. It is just like translating it into a Chinese book for them, not an English and Chinese text in combined. 

However, I have deleted those translated images. Sorry, I did not know that we're going to use the same for all the translations.

**Comment by JonathanSum** [comment]
I have deleted those translated images.

File: docs/zh/week06/06-2.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+重设门 $r_t$ 是用来决定忘记多少过去的信息。 在用来输入新「记忆」的部份 $\phi_h(W_hx_t + U_h(r_t\odot h_{t-1}) + b_h)$,，如果 $r_t$ 的系数为 0, ,那就没有保存任何过去的记忆。如果 $z_t$ 同时是0的话,那系统就完全重设。 ,因为 $h_t$ 只输入「现在的输入」。
+
+
+## 長期短期記憶(Long Short-Term Memory,简称LSTM)
+
+门控循环单元（GRU）是长短期记忆的简单版，长短期记忆是比门控循环单元早一点出的。 [Hochreiter, Schmidhuber, 1997](https://www.bioinf.jku.at/publications/older/2604.pdf). 一样地,长短期记忆也加入了记忆元件,它可以保持过去的信息。长短期记忆也是解决循环神经网络(RNN)长时间后会记忆丧失的问题。 LSTM的结构如下所示：
+
+<center>
+<img src="{{site.baseurl}}/images/week06/06-2/LSTM_zh.png" height="300px"/><br>
```
Comment (Atcold) [inline docs/zh/week06/06-2.md:173]: This link is pointing to a not existing file.

File: docs/zh/week06/06-2.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+
+## 循环神经网络(RNN)
+
+在卷积神经网络中，模組 之间的图形或他們的連接是沒有循环的。 卷积神经网络的模組 中是有顺序,就好像有由输入到输出一樣。
+
+
+
+<center>
+<img src="{{site.baseurl}}/images/week06/06-2/RNN_rolled_zh.png" /><br>
```
Comment (Atcold) [inline docs/zh/week06/06-2.md:25]: This file does not exist.

File: docs/zh/week06/06-2.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+ - $\text{Dec}(z(t))$: 生成输出的解码器
+
+
+## 循环网络：摊开循环的网络的循环
+
+摊开來的循环网络，他的输入是一个序列 $x_1, x_2, \cdots, x_T$.
+
+<center>
+ "
+<img src="{{site.baseurl}}/images/week06/06-2/RNN_unrolled_zh.png" /><br>
```
Comment (Atcold) [inline docs/zh/week06/06-2.md:45]: This file does not exist.

File: docs/zh/week06/06-2.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+
+虽然LSTM（长期短期记忆）在自然语言处理(NLP)中被广泛使用，但是它们的普及程度正在下降。例如，语音识别正朝着使用时间卷积网络(TCNN)的方向发展，而自然语言处理(NLP)正在朝着使用变型模型(Transformers)的方向发展。
+
+
+## 序列到序列模型
+
+提出的方法 [Sutskever NIPS 2014](https://papers.nips.cc/paper/5346-sequence-to-sequence-learning-with-neural-networks.pdf) 是第一个可以能和传统方法比较的神经机器翻译系统。它使用「编码器-解码器」体系结构，其中编码器和解码器均为多层LSTM。
+
+<center>
+<img src="{{site.baseurl}}/images/week06/06-2/Seq2Seq_zh.png" height="300px" /><br>
```
Comment (Atcold) [inline docs/zh/week06/06-2.md:199]: Why are there not existing files links?

**Comment by JonathanSum** [comment]
Changes request completed. I forgot to change it back. Those links were pointing to the translated image.

**Comment by Atcold** [comment]
Thanks a bunch!

**Pull request merged**